CMakeLists.txt

```diff
@@ -60,10 +60,12 @@ set(FlatHash_SRCS
 )
 
 set(FlatBuffers_Tests_SRCS
+  # FLATBUFFERS_TRACK_VERIFIER_BUFFER_SIZE macro based compilation does not happen
```

Comment:
You're right, that is actually pretty bad. My preferred fix would be to make that field always available. Feel free to do so as part of this PR or a separate one.